scapy/arch/windows/__init__.py

```diff
@@ -72,11 +72,40 @@ def _encapsulate_admin(cmd):
     # rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+def _hack_windows_subprocess(r=100):
+    """HACK: python 2.7 file descriptors.
+
+    This magic hack fixes https://bugs.python.org/issue19575
+    by suppressing passing HANDLE_FLAG_INHERIT to a range of
+    already opened file descriptors.
+    """
+    # See https://github.com/secdev/scapy/issues/1136
+    import stat
+    from ctypes import windll, wintypes
+    from msvcrt import get_osfhandle
+
+    HANDLE_FLAG_INHERIT = 0x00000001
+
+    for fd in range(r):
+        try:
+            s = os.fstat(fd)
+        except:
+            continue
+        if stat.S_ISREG(s.st_mode):
+            handle = wintypes.HANDLE(get_osfhandle(fd))
```

Comment:
Please conform to PEP-08, i.e. not '=' indenting.

Reply:
Cannot agree ;).
1. PEP8 it not absolute/religion
2. In this place the readability of the code has gone bad.

Reply:
I actually do agree with @guedou

Reply:
Any technical grounds? ;)
The main/primary goal is code readability (especially for peoples with 60+ with worse eyes. :),
not forcing conformance with 'absolute', especially if this 'absolute' speaks clearly
"""
Many projects have their own coding style guidelines.
In the event of any conflicts, such project-specific guides __take precedence__ for that project.
"""